### DISCUSSION TITLE
Wavy lines warnings in MD files

### DISCUSSION THREAD

**Discussion by tieniu1** [LazyVim/LazyVim#5164] [Q&A] [answered]
I'm a Neovim newbie and I'm loving LazyVim for helping me get started with Neovim for development work. Thank you for your hard work!

However, I've encountered some wavy lines warnings in my MD files and I'm not sure how to resolve them. I've searched through the issues and couldn't find a similar situation.

# Steps To Reproduce
1. Open any md file
2. Add something
```
# LazyVim

A starter template for [LazyVim](https://github.com/LazyVim/LazyVim).
Refer to the [documentation](https://lazyvim.github.io/installation) to get started.

# LazyVIm

#### 1. 什么是语义化标签

display

If you want to customize rules for a specific directory, you can create .markdownlint-cli2.yaml

```
### Expected Behavior

![Image](https://github.com/user-attachments/assets/f78b7a41-d0b4-4846-a830-a47bdf87e278)

I've tried checking the :messages and using K to hover over the text with wavy lines, but they don't provide any information about what's causing the warnings.

I haven't installed any other Markdown extensions, and whether I use extras with { import = "lazyvim.plugins.extras.lang.markdown" } or not, the wavy lines are still present.

I have a .markdownlint-cli2.yaml file in my project root directory, and I've verified that the rules are working through the command line. However, .markdownlint-cli2 shows no errors.

Can someone please help me out? Thanks!

**Comment by dpetka2001** [comment, marked as answer]
That comes from the spelling. Do `<leader>us` to disable spelling. For more permanent solution you would have to delete the autocommand that LazyVim has for spelling. It's in the docs if you search for it.

**Reply by tieniu1** [reply]
> That comes from the spelling. Do `<leader>us` to disable spelling. For more permanent solution you would have to delete the autocommand that LazyVim has for spelling. It's in the docs if you search for it.

Thank you, this solved my problem, I hope you have a wonderful day.
